tonic-reflection/tests/proto/mod.rs

```diff
@@ -1,337 +0,0 @@
-/// The message sent by the client when calling ServerReflectionInfo method.
```

Comment:
why were these deleted?

Reply:
Thought thats what you mean by https://github.com/hyperium/tonic/pull/622#discussion_r628253343.

Not sure what those files were for anyway. Removing them didn't break anything 🤔